src/vscode-bicep/icons/bicepparam-file.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24"><path d="M17.5 14H9l2-5h3.3l.7-1.5h-3L11 6l1-1.5h3L14.3 3H11L8.8 6l.9 1.5-6 7.5c-.4.5-.7 1.3-.7 2 0 1.5 1.1 2.8 2.5 3 0 0 10.5 1 12 1 1.9 0 3.5-1.6 3.5-3.5S19.4 14 17.5 14zM6 18.5c-.8 0-1.5-.7-1.5-1.5s.7-1.5 1.5-1.5 1.5.7 1.5 1.5-.7 1.5-1.5 1.5zm11.5 1c-1.1 0-2-.9-2-2s.9-2 2-2 2 .9 2 2-.9 2-2 2z" fill="#9f74b3"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18"><path d="M13.28 10.56H6.67l1.56-3.89h2.57l.54-1.17H9.01l-.78-1.17.78-1.17h2.33l-.54-1.17H8.23L6.52 4.32l.66 1.17-4.63 5.83c-.32.4-.54 1.01-.54 1.56 0 1.17.86 2.17 1.98 2.33-.01 0 8.13.78 9.29.78 1.5 0 2.72-1.22 2.72-2.72 0-1.5-1.22-2.72-2.72-2.72v.01Zm-8.84 3.5c-.64 0-1.17-.52-1.17-1.17 0-.65.52-1.17 1.17-1.17.65 0 1.17.52 1.17 1.17 0 .65-.52 1.17-1.17 1.17Zm8.94.78c-.86 0-1.56-.7-1.56-1.56 0-.86.7-1.56 1.56-1.56.86 0 1.56.7 1.56 1.56 0 .86-.7 1.56-1.56 1.56ZM1.2 18.01H0V0h1.2v18.01ZM3.2 0v1l-2 1V0h2Zm0 17v1h-2v-2l2 1ZM16.8 0H18v18h-1.2V0Zm0 0v2l-2-1V0h2Zm0 16v2h-2v-1l2-1Z" fill="#9F74B3"/></svg>
```

Comment:
Just curious where is the path generated from?

Reply:
https://www.freecodecamp.org/news/svg-basics-what-are-scalable-vector-graphics-and-how-do-you-use-them/ - I'm assuming the designer would have hand-crafted it.